CHANGELOG.md

```diff
@@ -3,28 +3,43 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.1.0 – Hebridean Black - [2023-mm-dd]
+## [[1.1.0](https://github.com/sanger-tol/readmapping/releases/tag/1.1.0)] – Hebridean Black - [2023-03-13]
 
-A cleaner and more efficient implementation of the pipeline.
+### Enhancements & fixes
 
-### `Added`
+- Bump minimum Nextflow version from `22.04.0` -> `22.10.1`
+- Updated pipeline template to [nf-core/tools 2.7.1](https://github.com/nf-core/tools/releases/tag/2.7.1)
+- Added improved resource settings
+- Added support for unit and full tests on Sanger HPC via Nextflow Tower
+- Added all unit test data on a S3 bucket
+- Added statistics subworkflow functionality to the alignment subworkflows
+- Removed support for iGenomes
+- Removed samtools faidx and minimap index module, it now happens on the fly
 
-– Updated pipeline template to [nf-core/tools v2.7.1](https://github.com/nf-core/tools/releases/tag/2.7.1)
-– Improved resource settings
-– Full and unit testing on Sanger farm directly from GitHub
+### Parameters
 
-### `Fixed`
+| Old parameter      | New parameter |
+| ------------------ | ------------- |
+| `--enable_conda`   |               |
+| `--minimap2_index` |               |
+| `--samtools_index` |               |
 
-– Genome indexing now happens on the fly
-– Most modules are now from nf-core instead of local
-– Merge statistics subworkflow functionality merged into the main alignment subworkflow
+> **NB:** Parameter has been **updated** if both old and new parameter information is present.
+> **NB:** Parameter has been **added** if just the new parameter information is present.
+> **NB:** Parameter has been **removed** if new parameter information isn't present.
 
-### `Dependencies`
+### Software dependencies
 
-### `Deprecated`
+Note, since the pipeline is now using Nextflow DSL2, each process will be run with its own [Biocontainer](https://biocontainers.pro/#/registry). This means that on occasion it is entirely possible for the pipeline to be using different versions of the same tool. However, the overall software dependency changes compared to the last release have been listed below for reference.
```

Comment:
```suggestion
Note, since the pipeline is using Nextflow DSL2, each process will be run with its own [Biocontainer](https://biocontainers.pro/#/registry). This means that on occasion it is entirely possible for the pipeline to be using different versions of the same tool. However, the overall software dependency changes compared to the last release have been listed below for reference.
```